docs/adr/0022-fix-double-vertical-scroll-bar-on-composer.md

```diff
@@ -0,0 +1,23 @@
+# 22. Fix double vertical scroll bar when hide status bar
+
+Date: 2023-04-21
+
+## Status
+
+- Issue: [#1740](https://github.com/linagora/tmail-flutter/issues/1740)
+
+## Context
+
+- Root cause: 
+    - When we using `html_editor_enhanced` lib can't auto resize `HTML Iframe` when changing the size of the screen
+    - `.note-editor.note-airframe .note-editing-area .note-editable,.note-editor.note-frame .note-editing-area .note-editable` has an overridden padding under the widgets
+
+## Decision
+
+- Use `setFullScreen` to `HTML Iframe` for the purpose of automatically resizing when there are changes
+- Remote padding of `.note-editor.note-airframe .note-editing-area .note-editable,.note-editor.note-frame .note-editing-area .note-editable`
```

Comment:
Should it be `Remove`?

Reply:
That padding distance is equal to the button bar, if you don't remove it, you will not be able to see the automatic line breaks.